backend/routes/sequence.js

```diff
@@ -1,5 +1,5 @@
 /**
- *  Copyright (C) 2019 3D Repo Ltd
+ *  Copyright (C) 2021 3D Repo Ltd
```

Comment:
shouldn't really be updating copy right notices?

i think if there's update it should say 2019-2021 if you want to pedantic about it 😆